developer-docs/server-installation/prerequisites.md

```diff
@@ -34,6 +34,25 @@ Before you start the installation process, ensure that you have the required inf
 | Other Services | 4core 16G 60G HDD | 1 |
 | Basic Load Balancers | - | 2 (Optional) |
 
+
+##### Automation on azure infrastructure creation

```

Comment:
Change to Heading 4: Automation on Azure Infrastructure Creation

Reply:
Done